components/Login/ForgotPassword.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { Form, Formik, Field } from 'formik'
+import { StyledTextInput } from '../FormStyles/InputBox'
+import * as Yup from 'yup'
+import {
+  BackButton,
+  EStyledButton,
+  Container,
+  Title,
+  SubTitle,
+} from '../Login/SignUp'
+import { RememberSignIn } from '../Login/SignIn'
+import styled from 'styled-components'
+import Link from 'next/link'
+
+export const TextButtonDiv = styled.div`
+  display: flex;
+  flex-flow: column;
+  justify-content: flex-end;
+  gap: 25px;
+`
+
```

Comment:
more comments would be nice (in general)

Reply:
We love purpose statements 😤